main/integration-tests/support/resources/base/resource.js

```diff
@@ -74,12 +74,13 @@ class Resource {
 
   async delete(params = {}, { api = this.api } = {}) {
     return this.doCall(async () => {
-      await this.axiosClient.delete(api, { params });
+      const response = this.axiosClient.delete(api, { params });
```

Comment:
please add 'await', because we want to make sure that this operation is completed successfully before we decide to remove the cleanup ttask